pkg/cloudprovider/aws/suite_test.go

```diff
@@ -144,9 +145,7 @@ var _ = Describe("Allocation", func() {
 				{SubnetId: aws.String("test-subnet-3"), AvailabilityZone: aws.String("test-zone-1c")},
 			}}
 			// Setup
-			pod := test.PodWith(test.PodOptions{
-				Conditions: []v1.PodCondition{{Type: v1.PodScheduled, Reason: v1.PodReasonUnschedulable, Status: v1.ConditionFalse}},
-			})
+			pod := pods.Pending()
```

Comment:
I'm sad that we lose the "test" concept due to the package rename. I guess we could do testpods, but that seems needless. Thoughts on something like
```
test.PodPending()
test.PodWith(test.PodOptions{})
```

Reply:
Seems very stuttery

Reply:
But I'll change it

Reply:
Changed!